lib/public/views/Environments/format/displayEnvironmentStatusHistoryOverview.js

```diff
@@ -25,13 +25,10 @@ export const displayEnvironmentStatusHistoryOverview = (environment) => {
 
     const statusHistory = historyItems
         .filter(({ status }) => status in STATUS_ACRONYMS)
-        .map(({ status }) => ({ status, content: STATUS_ACRONYMS[status] }));
+        .map(({ status }) => ({ status, acronym: STATUS_ACRONYMS[status] }));
 
     return h(
         '.flex-row',
-        statusHistory.map((value, index) => h('.flex-row', [
-            coloredEnvironmentStatusComponent(value.status, value.content),
-            h('', index < statusHistory.length - 1 ? '-' : ''),
-        ])),
+        environmentStatusHistoryComponent(statusHistory),
```

Comment:
I think it might be good to put something like
```
h(
        '.flex-row',
        h(
            '.w-25',
            environmentStatusHistoryComponent(statusHistory),
        ),
    );
```
in order to widen hooverable area, then you can have cursor beyond area of the content

Reply:
Besides, personally, tooltip on right hand side looks better - areas of the content and tooltip are not so close, but, as you prefer

Reply:
I really don't understand the reason of the `w-25` ? Why would you append 75% of blank space to the column?

Reply:
Well it might be w-15, w-50 whatever, I just wanted to suggest to widen area you can hoover in order to get tooltip, because with current solution, you need to hoover over status-items, which I consider not very user friendly, as well as having whole column "hooverable"

Reply:
[](url)
![Screenshot 2024-01-25 at 11 14 31](https://github.com/AliceO2Group/Bookkeeping/assets/65893715/a42e667b-a5db-4c50-9921-1df6effebc48)
![Screenshot 2024-01-25 at 11 13 00](https://github.com/AliceO2Group/Bookkeeping/assets/65893715/dfa4d73f-492a-47e9-9db2-ce8e572e8cb7)

Reply:
hmm, I didn't capture my cursor, but it's supposed to be next to left border of tooltip at upper screenshot

Reply:
Adding w-something will reduce the width of the cell's content to a percentage of the cell, which for me is a terrible idea because you force a white space at the end if the history item is long.
The point you raised is good though, I changed the way it's done by displaying an info icon at the top.